setup.py

```diff
@@ -99,7 +99,7 @@ def exec_file(path_segments):
     "isort==5.7.0",
     "black==20.8b1",
     "flake8-comprehensions",
-    "flake8-bugbear",
+    "flake8-bugbear==21.3.2",
```

Comment:
@callahad @ShadowJonathan: it's a shame that no attention was given to the comment above:

> We pin black so that our tests don't start failing on new releases.

This implies that we pin `flake8-bugbear` for some other reason?

Reply:
Where does that comment come from?

> This implies that we pin flake8-bugbear for some other reason?

We don't pin bugbear for any other reason, no.

Reply:
four lines above. Please remember to check the context around code you are changing.

Reply:
Ah, i didn't see that, i was mainly focused on just pinning bugbear, should an addition to the comment be made?

Reply:
it'd be nice. It may not be worth it. I'm mostly just asking that both you and @callahad remember to look for context around stuff you are changing/reviewing.